src/SectorConfiguration.tsx

```diff
@@ -132,12 +132,20 @@ export default observer(function SectorConfiguration() {
   const sectorArray = [sectorsForCurrent, sectorsForNext];
 
   const sectorChangeCountdown = timeToNextConfig <= 601;
-  const timelineRectangleHeight = document.querySelector('.timeline-rectangle0')?.clientHeight;
+  const timelineRectangleHeight = Number(document.querySelector('.accordion-body')?.clientHeight) - 20;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
```

Comment:
I will try to see next week why typescript is not happy there.

Reply:
I think it's just a trailing comment from something else I tested actually 🖌️